core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -44,6 +45,13 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
 
   public currentlyHoveredExecution: WorkflowExecutionsEntry | undefined;
 
+  // Pagination attributes
+  public currentPageIndex = defaultEnvironment.DEFAULT_PAGE_INDEX;
```

Comment:
please push the changes you made in the "default" config file, don't push the change of changing existing variables to `true`. Please also follow the same naming convention used. Lastly, add the variables types.